.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "tokenlandia-admin"
+    "default": "tokenlandia-admin-staging"
```

Comment:
You can have different projects in here and then select you want you want e.g.

```
"default": "tokenlandia-admin-staging",
"prod": "tokenlandia-admin",

Reply:
Just make sure inthe bash scripts it sets it e.g. `firebase use tokenlandia-admin-staging;`

Reply:
This is good to know. In fact I’ve since learned that the project you want to deploy to can be specified in the github action so I can revert this and use that instead. Both bash scripts choose the project they want to deploy to